ui/admin.py

```diff
@@ -31,6 +31,15 @@ class CollectionAdmin(admin.ModelAdmin):
     exclude = ('view_lists', 'admin_lists')
     date_hierarchy = 'created_at'
     readonly_fields = ['created_at']
+    search_fields = (
+        'title',
+        'slug',
+        'view_lists__name',
+        'admin_lists__name',
+        'stream_source',
```

Comment:
stream_source can only be one of 3 possible values, maybe this would be best included as a filter:

```python
list_filter = ['stream_source']
```